pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
building from source is reproducible if you check out an exact git commit or downloaded version, thats what we do elsewhere.